pkg/apis/core/validation/validation.go

```diff
@@ -3027,6 +3027,33 @@ func validateInitContainers(containers []core.Container, otherContainers []core.
 	return allErrs
 }
 
```

Comment:
nit: Can we explicitly say in the commit or add a comment saying which volumes are supported and checked for here?

Reply:
Giuseppe added this too